e2e/LongHaul/module/Parameters.cs

```diff
@@ -20,12 +20,19 @@ public enum JsonSerializingLibrary
 
     internal class Parameters
     {
+        [Option(
+            'g',
+            "GatewayHostName",
+            Required = true,
+            HelpText = "The gateway edge device to connects to.")]
+        public string GatewayHostName { get; set; }
```

Comment:
How about we make this optional and call out that if gateway host name is passed in then it is treated as an edge module, whereas if the gateway host isn't passed in then it is treated as IoT device module. That way the same test app can be used to test both scenarios.